tests/autopilot-capacity/aut_test.go

```diff
@@ -74,8 +75,31 @@ var ruleResizeBy50IfPvcCapacityLessThan10 = scheduler.AutopilotRuleParameters{
 	ExpectedPVCSize: 12079595520,
 }
 
+var ruleResizeBy50UntilPvcMaxSize20 = scheduler.AutopilotRuleParameters{
```

Comment:
Don't add a new rule test case. Add maxsize in the existing basic test case itself.

Reply:
Discussed offline. Would be good to have a separate test case.